walk-in-interview/src/main/java/com/google/firestore/FireStoreUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.google.firestore;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.Firestore;
+
+import com.google.configuration.TestFireStoreConfiguration;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.cloud.FirestoreClient;
+
+import java.io.IOException;
+
+/** Util methods related to Cloud Firestore database. */
+public final class FireStoreUtils {
+    private static Firestore firestore;
+
+    private FireStoreUtils() {}
+
+    // Use the application default credentials
+    private static void init() throws IOException {
+        GoogleCredentials credentials = GoogleCredentials.getApplicationDefault();
+        FirebaseOptions options = new FirebaseOptions.Builder()
+                .setCredentials(credentials)
+                .setProjectId(TestFireStoreConfiguration.PROJECT_ID)
```

Comment:
Instead of depending directly on TestFireStoreConfiguration, maybe create an ConfigurationFactory that can provides a FireStoreConfiguration. So when changing environment (dev, prod, test) we only need to inject different config factory at top level instead of changing different concrete class everywhere in the app?